HiP-App/HiP-Auth-v2
# Iss hipcms 617 basic auth functionality

How to test it:

* download the Identityserver4 samples from https://github.com/IdentityServer/IdentityServer4.Samples
* in the project "JsOidc" open the `app.js` file and set `client_id: "HiP-TokenGenerator"` and `scope: "openid profile email HiP-CmsWebApi"`
* open the `appsettings.Development.json` of the CmsWebApi project and set `"DOMAIN": "http://localhost:5001",`
* run the JsOidc application, the Auth-v2 service and the CmsWebApi service
* go to http://localhost:7101 and request an API token (when asked for the login data use the admin@hipapp.de user with our default password)
* in the next window you should get a token back, copy the token value and try a authenticated request (for example http://localhost:5000/Api/Annotation/Tags) to the CmsWebApi with the `Authorization` Header set to `Bearer [YOUR TOKEN HERE]`
* you should get a status code 200 back

Hope I did not forget anything
